<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>InfoComputers</class>
 <widget class="QDialog" name="InfoComputers">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>700</width>
    <height>700</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>681</width>
     <height>80</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>80</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(86, 197, 188);
image: url(:/Buttons/skil3-logo_80pxIN.png);
image-position: left;
color: rgb(255, 255, 255);
font: 18pt &quot;Agency FB&quot;;</string>
   </property>
   <property name="text">
    <string>              THE COMPUTING DATABASE</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>140</y>
     <width>68</width>
     <height>19</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 12pt &quot;AlternateGothic2 BT&quot;;
font: 12pt &quot;Agency FB&quot;;</string>
   </property>
   <property name="text">
    <string>Name:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>170</y>
     <width>68</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 12pt &quot;AlternateGothic2 BT&quot;;
font: 12pt &quot;Agency FB&quot;;</string>
   </property>
   <property name="text">
    <string>Type:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>220</y>
     <width>101</width>
     <height>19</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 12pt &quot;AlternateGothic2 BT&quot;;
font: 12pt &quot;Agency FB&quot;;</string>
   </property>
   <property name="text">
    <string>Was built?</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>250</y>
     <width>181</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 12pt &quot;AlternateGothic2 BT&quot;;
font: 12pt &quot;Agency FB&quot;;</string>
   </property>
   <property name="text">
    <string>Build/document year:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>300</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 12pt &quot;AlternateGothic2 BT&quot;;
font: 12pt &quot;Agency FB&quot;;</string>
   </property>
   <property name="text">
    <string>Linked to:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>410</y>
     <width>171</width>
     <height>19</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 12pt &quot;AlternateGothic2 BT&quot;;
font: 12pt &quot;Agency FB&quot;;</string>
   </property>
   <property name="text">
    <string>Short information:</string>
   </property>
  </widget>
  <widget class="QTextBrowser" name="sciLink">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>300</y>
     <width>301</width>
     <height>81</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextBrowser" name="infoBox">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>410</y>
     <width>301</width>
     <height>231</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="nameBox">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>140</y>
     <width>301</width>
     <height>25</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="typeBox">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>180</y>
     <width>301</width>
     <height>25</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="wbBox">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>220</y>
     <width>301</width>
     <height>25</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="byBox">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>260</y>
     <width>301</width>
     <height>25</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
